www/src/components/CodeBlock.jsx

```diff
@@ -59,6 +60,8 @@ function CodeBlock({ children, className, live }) {
             MiyazakiCard,
             HipsterIpsum,
             MenuIcon: ParagonIcons.Menu,
+            Image: children.includes('Dropzone') && typeof window !== 'undefined' ? window.Image : ParagonReact.Image,
```

Comment:
We already have `Image` component in the scope from Paragon components, but for `Dropzone`'s custom validation example we need native `Image` constructor to get uploaded image dimensions... This line will remove `Image` component from scope for `Dropzone` page.

Reply:
It is possible 😅, hasn't crossed my mind for some reason even though I used `window.Image` here...

updated, thanks!